rust/main/lander/src/adapter/chains/ethereum/nonce/state/core.rs

```diff
@@ -30,4 +30,8 @@ impl NonceManagerState {
             metrics,
         }
     }
+
+    pub fn metrics(&self) -> &EthereumAdapterMetrics {
+        &self.metrics
+    }
```

Comment:
Since `EthereumAdapter` will use `EthereumAdapterMetrics` directly, let's add it as a field to `EthereumAdapter` instead of exposing via `NonceManagerState`.